addons/web/static/src/search/filter_menu/custom_filter_item.js

```diff
@@ -257,15 +259,31 @@ export class CustomFilterItem extends Component {
                     [field.name, ">=", domainValue[0]],
                     [field.name, "<=", domainValue[1]]
                 );
+            } else if (operator.symbol === "today") {
```

Comment:
This is incorrect in general: we need to take care of timezones.

Reply:
just after `const descriptionArray = [field.string, operator.description.toString()];`
```js
if (operator.symbol === "today") {
        const domain =
          genericType === "date"
            ? `[("${field.name}", "=", context_today().strftime("%Y-%m-%d"))]`
            : `[
                        "&",
                            ("${field.name}", ">=", datetime.datetime.combine(context_today(), datetime.time(0,0,0)).to_utc().strftime("%Y-%m-%d %H:%M:%S")),
                            ("${field.name}", "<", datetime.datetime.combine(context_today() + relativedelta(days=1), datetime.time(0,0,0)).to_utc().strftime("%Y-%m-%d %H:%M:%S"))
                    ]`;
        return {
          description: descriptionArray.join(" "),
          domain,
          type: "filter",
        };
}
```

Reply:
should do what you want. Could you please adapt adapt your code and add 2 tests (one for each case)? If everything is fine it will land in 16.0. Thank you